src/library_glfw.js

```diff
@@ -1308,10 +1321,27 @@ var LibraryGLFW = {
         return false;
     },
 
+    /**
+     * CSS Scaling is a feature that is NOT part of the GLFW API, but for historical reasons, it is available
+     * in Emscripten.
+     * It is enabled by default but can be disabled by setting Module['EMSCRIPTEN_GLFW_DISABLE_CSS_SCALING'] to true.
+     * It is automatically disabled when using Hi DPI (the library overrides CSS sizes). */
+    isCSSScalingEnabled() {
+      return Module['EMSCRIPTEN_GLFW_DISABLE_CSS_SCALING'] !== true && !GLFW.isHiDPIAware();
```

Comment:
I'm somewhat loath to add more incoming module API elements like this.   Can we perhaps wait until somebody asks for this setting?

If we do end up adding it it would need to be added to INCOMING_MODULE_JS_API and then this line should be wrapped in `#if expectToReceiveOnModule('INCOMING_MODULE_JS_API')`.

I simpler more pay-as-you-go option could be to add some kind of new API for this either via `glfwSetWindowParam` or some new C API. 

I wonder if we can add a test for this behaviour?   Can we perhaps make a simple reftest that shows that the GLFW windows is scaled to the size of the canvas?  I'm not sure how hard that would be.